spec/factories/cities.rb

```diff
@@ -2,5 +2,11 @@
   factory :city do
     sequence(:name) {|i| "City Name #{i}" }
     state { create(:state) }
+
+    trait :with_holiday do
+      after :create do |city|
+        city.regional_holidays = create_list(:regional_holiday, 5)
```

Comment:
is this really creating the association in the db?

Reply:
It does. If I debug it and execute `city.regional_holidays` it returns all holidays associated with the city. I used the same trait on offices as inspiration.

Reply:
ok, just one last check, what happens if you do `city.reload.regional_holidays`

Reply:
It shows the associated cities. For a moment I thought I would have to manually save with `city.save` but it wasn't required. I've also tried `City.find(city.id).regional_holidays` and got the holidays as well. It seems like it's indeed persisted.

![image](https://user-images.githubusercontent.com/40892570/233991433-877a52be-b350-4abe-979d-04a20ccd5c2c.png)

Reply:
nice